src/Symfony/Component/Form/CHANGELOG.md

```diff
@@ -1,6 +1,12 @@
 CHANGELOG
 =========
 
+2.7.0
+-----
+
+ * deprecated AbstractType::setDefaultOptions() in favor of AbstractType::configureOptions().
```

Comment:
setDefaultOptions() has ben deprecated and the new method exists as preparation for the rename in 3.0.